### PR TITLE
Feature/scrutinizer

### PR DESCRIPTION
**Pull request by kallehauge**

### CODE REVIEW

**Comment by kallehauge** [comment]
Don't merge :warning:

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    "max-nested-callbacks": [1, 3],
+    "valid-jsdoc": [1, {
+      "prefer": {
+        "returns": "return",
+        "property": "prop"
+      },
+      "requireReturn": false
+    }]
+  }
+}
```
Comment (kasperg) [inline .eslintrc:97]: Why do we need this file when we have [`web/.eslintrc`](https://github.com/DBCDK/biblo-admin/blob/master/web/.eslintrc)?

File: .scrutinizer.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      - test
+      - theme
+    config:
+      standard: Drupal
+    filter:
+      paths: ['web/modules/custom/*', 'web/profiles/custom/*', 'web/themes/custom/*']
+  eslint:
+    use_native_config: true
+    filter:
+      paths: ['web/modules/custom/*', 'web/profiles/custom/*', 'web/themes/custom/*']
```
Comment (kasperg) [inline .scrutinizer.yml:18]: Do we need this filtering when we do not have third party code in our repo?

Comment (kallehauge) [inline .scrutinizer.yml:18]: Scrutinizer will build composer if there's a `composer.lock` file in the root

File: .eslintrc (hunk 1 above)
Comment (kallehauge) [inline .eslintrc:97]: Scrutinizer assumes that the `.eslintrc` file is in the root at the `.scrutinizer` file and I haven't found any ways to "point" to another file

File: .scrutinizer.yml (hunk 2 above)
Comment (kasperg) [inline .scrutinizer.yml:18]: :+1:

File: .eslintrc (hunk 1 above)
Comment (kasperg) [inline .eslintrc:97]: If they are equal then perhaps we could we note that somewhere?

Comment (kallehauge) [inline .eslintrc:97]: True. I'll add a section to the `README.md` file about Scrutinizer in general :relaxed:

Comment (kasperg) [inline .eslintrc:97]: Roger. When this is done I think this is ready to merge.

**Comment by kasperg** [comment]
Great. Merged in 3714cb8!

giphy

**Pull request closed without merging**